src/format.ts

```diff
@@ -115,101 +116,132 @@ function checkFormat(srcFiles: string[], baseArgs: string[], options: Options):
                           }
                         },
                         ['ignore', 'pipe', process.stderr])
-                    .stdout; 
+                    .stdout;
     out.setEncoding('utf8');
     out.on('data', (data: Buffer) => {
-      output += data; 
+      output += data;
     });
-    
-    out.on('end', () => {
-      findFormatErrorLines(output, options)
-        .then(() => {
-          resolve(output.indexOf('<replacement ') === -1 ? true : false);
-        });
+
+    out.on('end', async () => {
+      const files = output.split('<?xml version=\'1.0\'?>\n');
+      for (let i = 1; i < files.length; i++) {
+        const replacements = getReplacements(files[i]);
+        if (replacements.length > 0) {
+          const diff = await getDiffObj(srcFiles[i - 1], replacements)
```

Comment:
This line seems to be missing a `;`? Maybe you need to run `npm run fix`

Reply:
added